packages/components/src/components/tag/tag.tsx

```diff
@@ -18,9 +18,9 @@ import classNames from 'classnames';
 })
 export class Tag {
   /** (optional) Tag size */
-  @Prop() size?: string = '';
+  @Prop() size?: '' | 'small' = '';
   /** (optional) Tag variant */
-  @Prop() variant?: string = '';
+  @Prop() variant?: '' | 'secondary' = '';
```

Comment:
I think the same here - `?` makes the prop optional so I don't see why we need an empty string.

Reply:
Same here works fine like this.